interpreter/cling/include/cling/Interpreter/InvocationOptions.h

```diff
@@ -57,6 +57,11 @@ namespace cling {
     unsigned StdLib : 1;
     unsigned HasOutput : 1;
     unsigned Verbose : 1;
+    unsigned CxxModules : 1;
+    // The output path of any C++ PCMs we're building on demand.
+    std::string CachePath;
```

Comment:
Can you throw in what you're caching, I guess `ModuleCachePath`?

Reply:
Fixed, thanks!